lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -36,95 +36,95 @@ namespace ufe {
-		{
-			GfVec3f retValue;
-			if (op.Get<GfVec3f>(&retValue))
-			{
-				primXform.SetRotate(retValue);
-			}
-		}
-		// Scale
-		else if (opName == scale)
-		{
-			GfVec3f retValue;
-			if (op.Get<GfVec3f>(&retValue))
-			{
-				primXform.SetScale(retValue);
-			}
-		}
-		// Translate
-		else if (opName == trans)
-		{
-			GfVec3d retValue;
-			if (op.Get<GfVec3d>(&retValue))
-			{
-				primXform.SetTranslate(retValue);
-			}
-		}
-		// Scale / rotate pivot
-		else if (opName == pivot)
-		{
-			GfVec3f retValue;
-			if (op.Get<GfVec3f>(&retValue))
-			{
-				primXform.SetPivot(retValue);
-			}
-		}
-		// Scale / rotate pivot inverse
-		else if (opName == notPivot)
-		{
-			// automatically added, nothing to do.
-		}
-		// Not compatible
-		else
-		{
-			// Restore old
-			xformable.SetXformOpOrder(xformOps);
-			std::string err = TfStringPrintf("Incompatible xform op %s:", op.GetOpName().GetString().c_str());
-			throw std::runtime_error(err.c_str());
-		}
-	}
-	return primXform;
+        // RotateX, RotateY, RotateZ
+        if ((opName == rotX) || (opName == rotY) || (opName == rotZ))
+        {
+            float retValue;
+            if (op.Get<float>(&retValue))
+            {
+                if (opName == rotX)
+                    primXform.SetRotate(GfVec3f(retValue, 0, 0));
+                else if (opName == rotY)
+                    primXform.SetRotate(GfVec3f(0, retValue, 0));
+                else if (opName == rotZ)
+                    primXform.SetRotate(GfVec3f(0, 0, retValue));
+            }
+        }
+        // RotateXYZ
+        else if (opName == rotXYZ)
+        {
+            GfVec3f retValue;
+            if (op.Get<GfVec3f>(&retValue))
+            {
+                primXform.SetRotate(retValue);
+            }
+        }
+        // Scale
+        else if (opName == scale)
+        {
+            GfVec3f retValue;
+            if (op.Get<GfVec3f>(&retValue))
+            {
+                primXform.SetScale(retValue);
+            }
+        }
+        // Translate
+        else if (opName == trans)
+        {
+            GfVec3d retValue;
+            if (op.Get<GfVec3d>(&retValue))
+            {
+                primXform.SetTranslate(retValue);
+            }
+        }
+        // Scale / rotate pivot
+        else if (opName == pivot)
+        {
+            GfVec3f retValue;
+            if (op.Get<GfVec3f>(&retValue))
+            {
+                primXform.SetPivot(retValue);
+            }
+        }
+        // Scale / rotate pivot inverse
+        else if (opName == notPivot)
+        {
+            // automatically added, nothing to do.
+        }
+        // Not compatible
+        else
+        {
+            // Restore old
+            xformable.SetXformOpOrder(xformOps);
+            std::string err = TfStringPrintf("Incompatible xform op %s:", op.GetOpName().GetString().c_str());
+            throw std::runtime_error(err.c_str());
+        }
+    }
+    return primXform;
 }
```

Comment:
@ppt-adsk Converts Tabs to spaces. Apologies but I had to do it :) 

Github has a feature to ignore whitespace  changes. You can click the gear icon and choose Hide Whitespace changes:

![image](https://user-images.githubusercontent.com/48299423/88329346-cad27500-ccf7-11ea-92dc-6711f956aae6.png)